manifests/40-rbac.yaml

```diff
@@ -99,7 +99,14 @@ rules:
   verbs: ["*"]
 - apiGroups: ["operators.coreos.com"]
   resources: ["clusterserviceversions","operatorgroups","subscriptions"]
-  verbs: ["get","delete","list","update","watch"] 
+  verbs: ["get","delete","list","update","watch"]
+# Needed to check the feature gates
+- apiGroups: ["config.openshift.io"]
+  resources: ["clusterversions", "featuregates"]
+  verbs: ["list", "watch"]
+- apiGroups: ["apps"]
+  resources: ["replicasets"]
+  verbs: ["get"]
```

Comment:
what do we need replicasets for?

Reply:
> what do we need replicasets for?

For the event recorder. It detect the underline object automatically and I presume it's a replicaset because I got an error about prohibited access to that type of resource

Reply:
ok, this needs further investigation but it's not blocking now. NTO watches access already a bunch of objects and allowing `get` of replicasets is not too concerning.